pages/about.vue

```diff
@@ -43,6 +43,25 @@
         </v-col>
       </v-row>
 
+      <!-- <v-row class="d-flex justify-center iframe-container">
```

Comment:
(This is draft, so I assume you will end up removing this section? Thanks for all you've been doing to get all the new video content onto the website!)

Reply:
Yes!  Just removed it with a new commit.  Unless you have concerns, I think it's ok to merge to master.  If you see this and approve, go ahead and merge to master and DM me on Slack as a heads up plz!